services/fakes/fake_softlayer_product_package.go

```diff
@@ -0,0 +1,40 @@
+package test_helpers
+
+import (
+	"encoding/json"
+	"errors"
+
+	datatypes "github.com/maximilien/softlayer-go/data_types"
+	testhelpers "github.com/maximilien/softlayer-go/test_helpers"
+)
+
+type FakeProductPackageService struct{}
+
+func (fps *FakeProductPackageService) GetName() string {
+	return "Mock_Product_Package_Service"
+}
+
+func (fps *FakeProductPackageService) GetItemsByType(packageType string) ([]datatypes.SoftLayer_Product_Item, error) {
+	response, _ := testhelpers.ReadJsonTestFixtures("services", "SoftLayer_Product_Package_getItemsByType_virtual_server.json")
+
+	productItems := []datatypes.SoftLayer_Product_Item{}
+	json.Unmarshal(response, &productItems)
+
+	return productItems, nil
+}
+
+func (fps *FakeProductPackageService) GetItemPrices(packageId int, filters string) ([]datatypes.SoftLayer_Product_Item_Price, error) {
```

Comment:
Why are we not supporting these methods? Just don't include them in the interface, that's better than returning an error IMO